src/main/java/Zap.java

```diff
@@ -0,0 +1,252 @@
+        String userCommand;
+
+        do {
+            System.out.print("Enter a command: ");
+            userCommand = scanner.nextLine();
+
+            if (userCommand.equalsIgnoreCase("thank you and bye")) {
+                break;
+            } else if (userCommand.equalsIgnoreCase("hi")) {
+                System.out.println("Hello! I am ZAP and I am at your service!");
+            } else if (userCommand.equalsIgnoreCase("bye")) {
+                System.out.println("You should say thank you, then say bye.");
+            } else if (userCommand.equalsIgnoreCase("list")) {
+                displayTasks();
+            } else if (userCommand.startsWith("mark")) {
+                markTask(userCommand);
+            } else if (userCommand.startsWith("unmark")) {
+                unmarkTask(userCommand);
+            } else if (userCommand.startsWith("todo")) {
+                addTodoTask(userCommand);
+            } else if (userCommand.startsWith("deadline")) {
+                addDeadline(userCommand);
+            } else if (userCommand.startsWith("event")) {
+                addEvent(userCommand);
+            } else if (userCommand.startsWith("delete")) {
+                deleteTask(userCommand);
+            } else {
+                System.out.println("proper english pls. don't waste time already >:(");
+            }
+        } while (true);
+        scanner.close();
+    }
+
+    private static void addTodoTask(String userCommand) {
+
+        String taskDescription = userCommand.substring(4).trim();
+        if (taskDescription.isEmpty()) {
+            System.out.println("Woi. You think I robot then can waste my time. Gimme description >:(");
+        } else {
+            tasks.add(new TodoTask(taskDescription));
+
+            System.out.println("____________________________________________________________");
+            System.out.println(" Got it. Zappy boy added this task:");
+            System.out.println("   [T][ ]  " + taskDescription);
+            if (tasks.size() == 1) {
+                System.out.println(" Now you have 1 task in the list.");
+            } else {
+                System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+            }
+            System.out.println("____________________________________________________________");
+        }
+    }
+
+    private static void addDeadline(String userCommand) {
+        String[] descParts = userCommand.split("deadline");
+        String[] deadlineParts = descParts[1].split("/by", 2);
+
+        if (deadlineParts.length != 2) {
+            System.out.println("Invalid deadline format. Please use '/by' to specify day.");
+            return;
+        }
+
+        String description = deadlineParts[0];
+        String deadline = deadlineParts[1].trim();
+
+        tasks.add(new DeadlineTask(description, deadline));
+        System.out.println("____________________________________________________________");
```

Comment:
I noticed a lot of the same printing of this line here in your methods, maybe can extract this as a method and call it whenever you want to print this line